examples/notebooks/lithium-plating.ipynb

```diff
@@ -62,57 +64,88 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "A series of simple fast charging experiments based on those of Ren et al. [6] is defined here."
+    "A series of simple fast charging experiments based on those of Ren et al. [6] is defined here. We first initialise the model at 0% SoC by performing a C/20 discharge (see more details on how to initialise a model from a simulation in [this notebook](./initialise-model-with-solution.ipynb))."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<pybamm.models.full_battery_models.lithium_ion.dfn.DFN at 0x7fa1606dc190>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "# specify experiments\n",
     "pybamm.citations.register(\"Ren2018\")\n",
+    "\n",
+    "experiment_discharge = pybamm.Experiment(\n",
+    "    [\n",
+    "        \"Discharge at C/20 until 2.5 V (10 minute period)\",\n",
```

Comment:
could also add a 2.5V CV hold to really get it down to 0% SOC

Reply:
Given that we are doing a C/20 discharge (and thus that the stop threshold for CV should be below C/20) I don't think it would make a huge difference so I will leave it as it is (to match Simon's original notebook).

Reply:
makes sense